homeassistant/components/acaia/strings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "step": {
+      "user": {
```

Comment:
I think that bluetooth step is missing

Reply:
Mmh. Are you sure? Tests did not complain, and you never see the bluetooth step because you get redirected to user step immediately in the UI.

Reply:
no, I don't think so. 

- pretty sure tests would tell me nowadays
- I don't have it in lamarzocco
- I couldn't find it in other integrations